pkg/iris/api.go

```diff
@@ -67,7 +67,7 @@ func (a *api) GetConfiguration(configurationName string, accessToken string) ([]
 }
 
 func (a *api) PostResults(results []byte, accessToken string) error {
-	resultUrl := fmt.Sprintf("%s/services/vsm-iris/v1/results", a.uri)
+	resultUrl := fmt.Sprintf("%s/services/vsm-iris/v1/results/ecst", a.uri)
```

Comment:
any specific reason why you renamed the endpoint?

Reply:
the original plan was to make the connector backwards compatible with the old DTO, so that's why in Iris we currently have an  "old" endpoint which is the current results service and "ecst" to accommodate the new events.

Reply:
What is then the new plan ? Is it still to do that?

Reply:
Last time I checked, we said we will get rid anything related to the old once we can test the new eventhub e2e in the test env. Team Nimbus said they'll have it any time now but will need to confirm with Quentin again...

Reply:
so in theory we can revert it back to just /results once everything is finalised.